Package.swift

```diff
@@ -78,6 +79,13 @@ let package = Package(
                 "hash.txt",
                 "include/boringssl_prefix_symbols_nasm.inc",
                 "CMakeLists.txt",
+                /*
+                 * These files are excluded to support WASM which cannot include <netdb.h>.
```

Comment:
nit: it's WASI that provides these headers, not Wasm.

```suggestion
                 * These files are excluded to support WASI libc which doesn't provide <netdb.h>.
```